source/pool/contracts/Pool.sol

```diff
@@ -128,34 +128,53 @@ contract Pool is IPool, Ownable2Step {
   /**
    * @notice Set claims from previous pool contract
    * @dev Only owner
-   * @param _root bytes32
+   * @param _groupId bytes32
    * @param _accounts address[]
    */
   function setClaimed(
-    bytes32 _root,
+    bytes32 _groupId,
     address[] memory _accounts
   ) external override multiAdmin {
-    if (roots[_root] == false) {
-      roots[_root] = true;
-    }
+    // if (roots[rootsByGroupId[_groupId]] == false) {
+    //   roots[rootsByGroupId[_groupId]] = true;
+    // }
     for (uint256 i = 0; i < _accounts.length; i++) {
       address account = _accounts[i];
-      if (claimed[_root][account]) revert AlreadyClaimed();
-      claimed[_root][account] = true;
+      if (claimed[rootsByGroupId[_groupId]][account]) revert AlreadyClaimed();
+      claimed[rootsByGroupId[_groupId]][account] = true;
     }
-    emit Enable(_root);
+    emit Enable(rootsByGroupId[_groupId]);
   }
 
   /**
    * @notice Enables claims for a merkle tree of a set of scores
    * @param _root bytes32
    */
-  function enable(bytes32 _root) external override multiAdmin {
-    if (roots[_root]) revert RootExists(_root);
-    roots[_root] = true;
+  function enable(
+    bytes32 _groupId,
+    bytes32 _root
+  ) external override multiAdmin {
+    if (rootsByGroupId[_groupId] != 0) revert GroupIdExists(_groupId);
```

Comment:
Question for @dmosites whether setting the root should be permanent. 
My feeling is that it should be possible to overwrite a root, because it's something that's quite hard to verify when the transaction is submitted, so I can see errors occurring with this...